tests/integration/tier0/backup-release/test_backup_release.py

```diff
@@ -41,32 +183,49 @@ def test_backup_os_release_no_envar(shell, convert2rhel):
         "-y --no-rpm-va -k {} -o {} --debug --keep-rhsm".format(
             env.str("SATELLITE_KEY"),
             env.str("SATELLITE_ORG"),
-        )
+        ),
+        unregister=True,
     ) as c2r:
         c2r.expect("set the environment variable 'CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK.")
         assert c2r.exitstatus != 0
 
     assert shell("find /etc/os-release").returncode == 0
 
+    # Restore system
+    teardown_repositories_and_cleanup(shell)
+    del os.environ["CONVERT2RHEL_UNSUPPORTED_SKIP_KERNEL_CURRENCY_CHECK"]
 
+
+@pytest.mark.with_envar
 def test_backup_os_release_with_envar(shell, convert2rhel):
     """
     In this scenario the variable `CONVERT2RHEL_UNSUPPORTED_INCOMPLETE_ROLLBACK` is set.
     This test case removes all the repos on the system and validates that
     the /etc/os-release package is being backed up and restored during rollback.
-    Ref ticket: OAMG-5457. Note that after the test, the $releaserver
+    Ref ticket: OAMG-5457. Note that after the test, the $releasever
```

Comment:
Changing to redirect
```suggestion
    Ref ticket: RHELC-51. Note that after the test, the $releasever
```